### ISSUE TITLE
How can I use these networks?

### ISSUE THREAD

**Issue by bclman** [lightvector/KataGo#372]
https://katagotraining.org/networks/

katago v1.7.0 opencl version
rtx 3090

I tried new stronger network file.
model unsupported version 10 error

**Comment by lightvector** [comment]
Right, there is no release yet that supports them. There should be a release in within the next week or two, if you are in a hurry to try them out you can also compile the master branch yourself. :)

**Comment by l1t1** [comment]
could you upload a compiled version now for conviene and when will the project open for contribute? @lightvector

**Comment by portkata** [comment]
> https://katagotraining.org/networks/
> 
> katago v1.7.0 opencl version
> rtx 3090
> 
> I tried new stronger network file.
> model unsupported version 10 error

also if you have android phone/tablet, you can use this app https://github.com/aki65/aki65.github.io/releases
